src/widgets/dropdown/widget-dropdown-view.js

```diff
@@ -68,38 +71,41 @@ module.exports = cdb.core.View.extend({
 
   _onGlobalClick: function (ev) {
     if (this._$target.get(0) !== $(ev.target).closest(this._$target).get(0)) {
-      this.model.set('open', false);
+      this.model.set('widget_dropdown_open', false);
     }
   },
 
   _onKeyUp: function (ev) {
     if (ev.keyCode === 27) {
-      this.model.set('open', false);
+      this.model.set('widget_dropdown_open', false);
       return false;
     }
   },
 
   _onChangeOpen: function () {
-    if (this.model.get('open')) {
+    if (this.model.get('widget_dropdown_open')) {
       this._open();
     } else {
       this._close();
     }
   },
 
-  _pin: function () {
-    this.model.set('open', false);
-    this.trigger('click', 'pin');
+  _togglePinned: function () {
+    var pinned = !this.model.get('pinned');
+    this.$('.js-inputPinned').attr('checked', pinned);
+    this.model.set('pinned', pinned);
   },
 
-  _toggle: function () {
-    this.model.set('open', false);
-    this.trigger('click', 'toggle');
+  _toggleCollapsed: function () {
+    var collapsed = !this.model.get('collapsed');
```

Comment:
So we don't need this thing anymore, because render will display checked as it should be.